maths/krishnamurthy_number.py

```diff
@@ -0,0 +1,49 @@
+"""
+ == Krishnamurthy Number ==
+It is also known as Peterson Number
+A Krishnamurthy Number is a number whose sum of the
+factorial of the digits equals to the original
+number itself.
+
+For example: 145 = 1! + 4! + 5!
+    So, 145 is a Krishnamurthy Number
+"""
+
+
+def factorial(digit: int) -> int:
```

Comment:
@Utsav1999 Kindly also add docstrings to your function

Reply:
Or `from math import factorial`